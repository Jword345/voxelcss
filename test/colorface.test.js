'use strict';

const ColorFace = require('../lib/ColorFace');

describe('ColorFace', function() {
    let face;
    beforeEach(() => {
        face = new ColorFace('#333');
    });
    it('can get hex and RGBA color values', function() {
        expect(face.getHex()).toMatchSnapshot();
        expect(face.getColor()).toMatchSnapshot();
    });
});
