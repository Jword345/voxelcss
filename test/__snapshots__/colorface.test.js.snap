// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorFace can get hex and RGBA color values 1`] = `"333333"`;

exports[`ColorFace can get hex and RGBA color values 2`] = `
Object {
  "a": 1,
  "b": 51,
  "g": 51,
  "r": 51,
}
`;
